apps/web/pages/[user].tsx

```diff
@@ -225,20 +239,31 @@ export type UserPageProps = {
 
 export const getServerSideProps: GetServerSideProps<UserPageProps> = async (context) => {
   const ssr = await ssrInit(context);
-  const { currentOrgDomain, isValidOrgDomain } = orgDomainConfig(context.req.headers.host ?? "");
-
+  const { currentOrgDomain, isValidOrgDomain } = orgDomainConfig(
+    context.req.headers.host ?? "",
+    context.query.orgSlug
```

Comment:
Making sure we respect the way of accessing an org link using `/org/orgSlug` as well

Reply:
Also, using query.orgSlug might allow a non-org booker URL to server the org booking page by passing `orgSlug` query param.

We should probably use context.params here which has just dynamic params and no additional query param from client.